src/utils/rss/generate-feed.js

```diff
@@ -1,9 +1,12 @@
 const cheerio = require(`cheerio`)
 const tagsHelper = require(`@tryghost/helpers`).tags
 const _ = require(`lodash`)
+const config = require(`../../config/siteConfig`)
+
 
 const generateItem = function generateItem(post) {
   const itemUrl = post.canonical_url || post.url
+  const itemFinalUrl= itemUrl.replace("https://admin.mayadata.io/", `${config.siteUrl}/blog/`);
```

Comment:
remove line 8 and 9

and use this instead

```
const itemUrl = `${siteConfig.siteUrl}/blog/${post.slug}/`;
```

Reply:
![image](https://user-images.githubusercontent.com/72143117/104167617-a6439400-5422-11eb-8c93-c76c4cd83280.png)

those two things are different @IsAmrish still I need to make a link as you told me above?

Reply:
You are comparing two different objects.